tests/test_app/templates/Error/error500.php

```diff
@@ -1,6 +1,8 @@
 <?php
 use Cake\Core\Configure;
 use Cake\Error\Debugger;
+use function \Cake\Core\h;
+use function \Cake\I18n\__d;
```

Comment:
Is `h` & `__d` functions required? (Also for error400.php)

Reply:
Those funcitons are used in those templates but not really nessesary.